copier.yml

```diff
@@ -20,4 +20,10 @@ nightly_deps:
   help: |
     List of dependencies to install from Git during nightly builds,
     separated by commas. Make sure to also list transitive dependencies,
-    otherwise their released versions will be used.
\ No newline at end of file
+    otherwise their released versions will be used.
+related_projects:
+  type: str
+  help: |
+    List of related projects, separated by commas.
+    This will be used to generate a list of links in the documentation.
+    For example, "Scipp,Sciline,Plopp".
```

Comment:
When I added it in plopp, I could not decide whether the names should start with an uppercase letter or not...

Reply:
You mean the theme changes the case?